core/domain/exp_domain_test.py

```diff
@@ -752,7 +754,8 @@ def test_validation(self):
             'training_data': [],
             'tagged_skill_misconception_id': 1
         }
-        init_state.update_interaction_answer_groups([answer_groups_dict])
+        init_state.update_interaction_answer_groups(
+            state_domain.AnswerGroup.from_dict(answer_groups_dict))
```

Comment:
Ditto here and elsewhere*

Reply:
`answer_groups_dict = {` I have changed here too, Did i do anything wrong.

Reply:
`state_domain.AnswerGroup.from_dict` accepts dict whereas `answer_groups_dict` is a list right?

Reply:
I have changed answer_groups_dict from list to a dict
 ` answer_groups_dict = {
            'outcome': {
                'dest': exploration.init_state_name,
                'feedback': {
                    'content_id': 'feedback_1',
                    'html': 'Feedback'
                },
                'labelled_as_correct': False,
                'param_changes': [],
                'refresher_exploration_id': None,
                'missing_prerequisite_skill_id': None
            },
            'rule_specs': [{
                'inputs': {
                    'x': 'Test'
                },
                'rule_type': 'Contains'
            }],
            'training_data': [],
            'tagged_skill_misconception_id': 1
        }`